### PR TITLE
Add is not null to query builder

### LINKED ISSUE #293: Add IS NOT NULL to query builder

### PR DESCRIPTION
**Pull request by imilinovic**
### Description

Add IS NOT to query builder.

### Pull request type

Please delete options that are not relevant.

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring with functional or API changes
- [ ] Refactoring without functional or API changes
- [ ] Build or packaging related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### Related issues
closes #293 

Delete section if this PR doesn't resolve any issues. 

Closes (link to issue)

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

######################################

### Reviewer checklist (the reviewer checks this part)
- [ ] Core feature implementation
- [ ] Tests
- [ ] Code documentation
- [ ] Documentation on [gqlalchemy/docs](https://github.com/memgraph/gqlalchemy/tree/main/docs)

######################################

### CODE REVIEW

**Review by antepusic — APPROVED** [review]
LGTM

File: gqlalchemy/query_builders/declarative_base.py [hunk 1]
```diff
@@ -169,6 +175,7 @@ class WhereConditionPartialQuery(PartialQuery):
     def __init__(self, item: str, operator: Operator, keyword: Where = Where.WHERE, is_negated: bool = False, **kwargs):
         super().__init__(type=keyword.name if not is_negated else f"{keyword.name} {Where.NOT.name}")
         self.query = self._build_where_query(item=item, operator=operator, **kwargs)
+        print(self.construct_query())
```
Comment (katarinasupe) [inline gqlalchemy/query_builders/declarative_base.py:178]: Maybe remove this print?

**Review by katarinasupe — APPROVED** [review]
All looks fine. As soon as deda fixes the workflow, the tests should pass as well.

**Comment by katarinasupe** [comment]
@imilinovic merge the latest main for tests to pass.
Once the tests pass, feel free to squash and merge.

**Comment by katarinasupe** [comment]
I extended the description in #293 which better explains what needs to be done. I am closing this PR since it is not fixing that issue.

**Pull request closed without merging**